github-bot/harvester_github_bot/route.py

```diff
@@ -0,0 +1,55 @@
+import http
+
+from flask import request
+from flask_httpauth import HTTPBasicAuth
+from werkzeug.security import check_password_hash
+
+from harvester_github_bot.config import app, FLASK_USERNAME, FLASK_PASSWORD, GITHUB_OWNER, GITHUB_REPOSITORY
+from harvester_github_bot.issue_transfer import issue_transfer
+from harvester_github_bot.backport import backport
+
+auth = HTTPBasicAuth()
+
+
+@auth.verify_password
+def verify_password(username, password):
+    if check_password_hash(FLASK_USERNAME, username) and check_password_hash(FLASK_PASSWORD, password):
+        return username
+
+
+@app.route('/zenhub', methods=['POST'])
+@auth.login_required
+def zenhub():
+    form = request.form
+    organization = form.get('organization')
+    repo = form.get('repo')
```

Comment:
has a same name with the global variable repo in `harvester_github_bot.config`, It's better to use other names?